### DISCUSSION TITLE
Enable email notifications

### DISCUSSION THREAD

**Discussion by j0sm3n** [ellite/Wallos#245] [Q&A] [answered]
I'm trying to set up email notifications, with a Gmail account, but I don't know what data I have to enter in each field. Could someone give me a hand? Thank you in advance

**Comment by ellite** [comment, marked as answer]
Hi,

SMTP Address: smtp.gmail.com
Port: 587
TLS
SMTP Username: your email address
SMTP Password: You need to use app password (https://support.google.com/mail/answer/185833?hl=en)
From email (optional): Can be your email address

Regards.

**Reply by j0sm3n** [reply]
I was trying to set the account password, instead of creating an application password. Thank you
